### PR TITLE
delete user seems to be functioning properly

### PR DESCRIPTION
**Pull request by pvmend**
Delete user succesfully executes and redirects user to the home page

### CODE REVIEW

File: client/src/pages/AccountInfo.jsx [hunk 1]
```diff
@@ -53,7 +56,7 @@ const handleFormSubmit = async (event ) => {
             variables: { ...userFormData, id: Auth.getProfile().data._id },
         });
         const { token } = response.data.updateUser;
```
Comment (KateMcRo) [inline client/src/pages/AccountInfo.jsx:58]: This line can be deleted as well.

File: client/src/pages/AccountInfo.jsx [hunk 2]
```diff
@@ -133,30 +136,28 @@ return (
...
-              });
-            }
-          }>
-            Delete Profile
-          </Button>
-
-          {/* <Box sx={{ textAlign: "end", alignSelf: "end" }}>
-            <NavLink to="/signup">Create an account. Sign up</NavLink>
-          </Box> */}
+          <Button type="submit" variant="contained" sx={{ width: "100%" }} onClick={async (event) => {
```
Comment (KateMcRo) [inline client/src/pages/AccountInfo.jsx:139]: Please move this into its own function named handleDeleteUser or something.

**Review by KateMcRo — CHANGES_REQUESTED** [review]
Please see comments.

File: client/src/pages/AccountInfo.jsx (hunk 1 above)
Comment (pvmend) [reply client/src/pages/AccountInfo.jsx:58]: 56? or 58?

Comment (KateMcRo) [reply client/src/pages/AccountInfo.jsx:58]: const { token } = response.data.updateUser;

**Review by KateMcRo — APPROVED** [review]

**Pull request merged**